package.json

```diff
@@ -76,10 +77,6 @@
     "jest": "29",
     "jest-environment-jsdom": "29.7.0"
   },
-  "optionalDependencies": {
-    "@adobe/adobe-intl-aladdin": "latest",
-    "@adobe/adobe-intl-harvester": "latest"
-  },
```

Comment:
Please add optional dependencies back and regenerate package-lock.json

Reply:
I realized that adding those packages causes issues on dev machines. Those dependencies are from Adobe's private repo, while others are from a public repo. `npm` doesn't allow mapping a scope to multiple repos in `npmrc`.